addOns/automation/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add job to configure the active scanner, `activeScan-config`.
 - Allow to enable/disable jobs (Issue 5845).
 - Method to allow the user to set the exit code via a script.
+- Add job exitStatus (Issue #6928)
```

Comment:
I'd suggest switching the words around so that it's obvious that it's a job not a status per job.